frontend/src/metabase/collections/components/PinnedQuestionCard/PinnedQuestionCard.unit.spec.tsx

```diff
@@ -24,7 +24,7 @@ function setup() {
   });
   const collectionItem = createMockCollectionItem({
     id: card.id,
-    type: "card",
+    model: "card",
```

Comment:
This change doesn't make a huge impact on the test since the `model` is already defaulted to `card`
https://github.com/metabase/metabase/blob/6634abddfcd0327565e67c2124b55601a823d48c/frontend/src/metabase-types/api/mocks/collection.ts#L20